scripts/support/create-support-bundle.sh

```diff
@@ -0,0 +1,313 @@
+INDENT_WIDTH='  '
+indent=""
+
+collector_config=/opt/observiq-otel-collector/config.yaml
+
+# Colors
+num_colors=$(tput colors 2>/dev/null)
+if test -n "$num_colors" && test "$num_colors" -ge 8; then
+  reset="$(tput sgr0)"
+  fg_cyan="$(tput setaf 6)"
+  fg_green="$(tput setaf 2)"
+  fg_red="$(tput setaf 1)"
+  fg_yellow="$(tput setaf 3)"
+fi
+
+if [ -z "$reset" ]; then
+  sed_ignore=''
+else
+  sed_ignore="/^[$reset]+$/!"
+fi
+
+printf() {
+  if command -v sed >/dev/null; then
+    command printf -- "$@" | sed -E "$sed_ignore s/^/$indent/g"  # Ignore sole reset characters if defined
+  else
+    # Ignore $* suggestion as this breaks the output
+    # shellcheck disable=SC2145
+    command printf -- "$indent$@"
+  fi
+}
+
+increase_indent() { indent="$INDENT_WIDTH$indent" ; }
+decrease_indent() { indent="${indent#*"$INDENT_WIDTH"}" ; }
+
+# Color functions reset only when given an argument
+# Ignore "parameters are never passed"
+# shellcheck disable=SC2120
+reset() { command printf "$reset$*$(if [ -n "$1" ]; then command printf "$reset"; fi)" ; }
+fg_cyan() { command printf "$fg_cyan$*$(if [ -n "$1" ]; then command printf "$reset"; fi)" ; }
+fg_green() { command printf "$fg_green$*$(if [ -n "$1" ]; then command printf "$reset"; fi)" ; }
+fg_red() { command printf "$fg_red$*$(if [ -n "$1" ]; then command printf "$reset"; fi)" ; }
+fg_yellow() { command printf "$fg_yellow$*$(if [ -n "$1" ]; then command printf "$reset"; fi)" ; }
+
+# Intentionally using variables in format string
+# shellcheck disable=SC2059
+info() { printf "$*\\n" ; }
+
+# Intentionally using variables in format string
+# shellcheck disable=SC2059
+error() {
+  increase_indent
+  printf "$fg_red$*$reset\\n"
+  decrease_indent
+}
+
+# Intentionally using variables in format string
+# shellcheck disable=SC2059
+success() { printf "$fg_green$*$reset\\n" ; }
+
+observiq_banner()
+{
+  fg_cyan "           888                                        8888888 .d88888b.\\n"
+  fg_cyan "           888                                          888  d88P\" \"Y88b\\n"
+  fg_cyan "           888                                          888  888     888\\n"
+  fg_cyan "   .d88b.  88888b.  .d8888b   .d88b.  888d888 888  888  888  888     888\\n"
+  fg_cyan "  d88\"\"88b 888 \"88b 88K      d8P  Y8b 888P\"   888  888  888  888     888\\n"
+  fg_cyan "  888  888 888  888 \"Y8888b. 88888888 888     Y88  88P  888  888 Y8b 888\\n"
+  fg_cyan "  Y88..88P 888 d88P      X88 Y8b.     888      Y8bd8P   888  Y88b.Y8b88P\\n"
+  fg_cyan "   \"Y88P\"  88888P\"   88888P'  \"Y8888  888       Y88P  8888888 \"Y888888\"\\n"
+  fg_cyan "                                                                   Y8b  \\n"
+
+  reset
+}
+
+separator() { printf "===================================================\\n" ; }
+
+banner() {
+  printf "\\n"
+  separator
+  printf "| %s\\n" "$*" ;
+  separator
+}
+
+usage() {
+  increase_indent
+  USAGE=$(cat <<EOF
+Usage:
+  Collects support bundle for observIQ Agent
```

Comment:
We should keep the verbiage like this as much as possible, to avoid violating the CNCF's terms.
```suggestion
  Collects support bundle for observIQ Distro for OpenTelemetry Collector
```